lib/datasource/npm/get.ts

```diff
@@ -174,6 +174,9 @@ export async function getDependency(
         extraBaseUrls,
       });
     }
+    if (sourceUrl === undefined) {
+      sourceUrl = res.repository.url;
```

Comment:
I think if we are doing sourceUrl parsing in the datasource/index then we can remove the custom parsing from this npm datasource and just return the raw url.

Reply:
Done.